<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1143</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidgetMain">
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1143</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionOpenBase"/>
    <addaction name="actionCloseBase"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionNewBase"/>
    <addaction name="actionExit"/>
    <addaction name="separator"/>
    <addaction name="actionFile01"/>
    <addaction name="actionFile02"/>
    <addaction name="actionFile03"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenPayslip"/>
  </widget>
  <action name="actionCloseBase">
   <property name="text">
    <string>CloseBase</string>
   </property>
   <property name="toolTip">
    <string>Зактрыть базу</string>
   </property>
  </action>
  <action name="actionOpenBase">
   <property name="text">
    <string>OpenBase</string>
   </property>
  </action>
  <action name="actionFile01">
   <property name="text">
    <string/>
   </property>
  </action>
  <action name="actionFile02">
   <property name="text">
    <string/>
   </property>
  </action>
  <action name="actionFile03">
   <property name="text">
    <string/>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>SaveAs</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionNewBase">
   <property name="text">
    <string>NewBase</string>
   </property>
  </action>
  <action name="actionOpenPayslip">
   <property name="text">
    <string>Ведомость</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
